File: Java/Week 6/Recursion/CountDigits.java
class CountDigits {

static int countDigits(int n){
    if(n==0){
        return 0;
    }
    return 1 + countDigits(n/10);
}


    public static void main(String[] args) {
        System.out.println(sumOfDigits(300403));
    }
}

File: Cpp/Week 6/Recursion/countDigits.cpp
// Count the number of digits in a number recursively

int countDigits(int n){
    if(n==0){
        return 0;
    }
    return 1 + countDigits(n/10);
}

int main() {
    cout<<countDigits(300409);
    return 0;
}
